d3a_api_client/redis_client_base.py

```diff
@@ -32,14 +26,12 @@ def wrapped(self, *args, **kwargs):
 
 
 class RedisClient(APIClientInterface):
```

Comment:
I would suggest to rename this to `RedisClientBase` to have the same name of the module (makes it easier to search in the project).

Reply:
I agree! Will do it!

Reply:
Actually, I dont see the need that these should be two classes. The potential `RedisClientBase` will not be used anywhere else. (only in auto_offer_bid_on_every_market.py and d3a_api_client/setups/auto_offer_bid_on_device.py which I will deleted also because there are not needed any more) 
EDIT: Actually, the future `RedisClientBase` could be reused in `RedisMarketClient`, too. OI wonder how this wasn't done before. I will probably see when implementing.

Reply:
When you say "two classes" you mean `RedisClient` and `RedisDeviceClient`, right? If that's the case, I agree that refactoring them into one class seems to be good give our current use.

Reply:
I am currently trying to have one Baseclass `RedisClientBase` that works as Superclass for both `RedisDeviceClient` and `RedisMarketClient`. This would be the most clear way IMO.

Reply:
Ah ok! Now I get it 👍